lhotse/kaldi.py

```diff
@@ -80,6 +80,7 @@ def load_kaldi_data_dir(
     map_string_to_underscores: Optional[str] = None,
     use_reco2dur: bool = True,
     num_jobs: int = 1,
+    feature_type: str = 'kaldi-fbank'
```

Comment:
You also need to update
https://github.com/lhotse-speech/lhotse/blob/cb0b7d341fa348063c6925e5e2512a65c33206a4/lhotse/bin/modes/kaldi.py#L35-L41

You can add another option for `feature_type`.

Reply:
You forgot to update the subsequent python function.

You have to add it as an argument for the function that follows.

Reply:
Sorry for my carelessness, see if any further modifications need to be done.